packit_service/service/views.py

```diff
@@ -72,35 +65,23 @@ def _get_build_logs_for_build(
     else:
         title_identifier = ""
 
-    response = (
-        "<html><head>"
-        f"<title>{build_description} {project.namespace}/{project.repo_name}:"
-        f" {title_identifier}</title></head><body>"
-        f"{build_description} ID: {build.build_id}<br>"
-        f"Submitted: {models.optional_time(build.build_submitted_time)}<br>"
-        f"State: {build.status}<br><br>"
-        f'Build web interface URL: <a href="{build.web_url}">{build.web_url}</a><br>'
+    return render_template(
+        "build_logs.html",
```

Comment:
With this change it seems to me this view is now build_info rather than build_logs - would it make sense to rename it?

Reply:
Definitely :+1:

Reply:
This requires changing some parts of the routing too, to make it consistent all over the place.

Next Friday.

Reply:
or we can merge this as it is, and do the change as a new PR next friday :)

Reply:
Here is proposal to make this consistent all over in `../service/views.py`:

- rename `build_logs.html` to `build_info.html` as you suggested
- rename and redirect routes from `/copr-build/<id>/logs` to `/copr-build/<id>`. The same for `/koji-build`
- rename function names to match the above

What do you think?

Reply:
SGTM